practice/00-HTML-CSS-basics/topic0.1iv.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+  <title>Lists</title>
+ </head>
+ <body>
+  <h1>Lists of expenses</h1>
+   <ol>
```

Comment:
If the list doesn't have a particular order, you should use ul (unordered list) instead of ol (ordered list)